Python/Product/PythonTools/PythonTools/Debugger/DebugLaunchHelper.cs

```diff
@@ -158,9 +158,25 @@ static class DebugLaunchHelper {
             // jsonObj["$adapterArgs"] = "";
             // jsonObj["$adapterRuntime"] = "";
 
+            // There is a bug when trying to run the 3.10+ python.exe directly from conda environments
+            // See https://github.com/conda/conda/issues/11072
+            // The workaround is to run this instead:
+            // conda run -n <env> --no-capture-output --live-stream python <debugpy>
+            //if (CondaUtils.IsCondaEnvironment(Path.GetDirectoryName(config.GetInterpreterPath())) &&
+            //    config.Interpreter.Version > new Version(3, 10)) {
+            //
+            //    var condaProvider = provider.GetComponentModel().GetService<CondaEnvironmentFactoryProvider>();
+            //    var condaExe = condaProvider
+            //
+            //    //DoConda310Workaround(provider, jsonObj);
```

Comment:
is this all commented out on purpose?

Reply:
Ah that was stuff I meant to remove. I'll remove it :)